.github/workflows/pip_release.yaml

```diff
@@ -0,0 +1,17 @@
+name: pip release
+on:
+  release:
+    types: [published]
+jobs:
+  pypi:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - run: python3 -m pip install --upgrade build && python3 -m build
```

Comment:
not picking a version here might bite you eventually but i guess you did that on purpose (better to fail/fix than go stale?)

Reply:
@earonesty that's exactly what I thought 😊